ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/domain/CaseEvent.java

```diff
@@ -2,6 +2,7 @@
 
 public enum CaseEvent {
 
+    CREATE_NEW_CASE("IssueClaim"),
```

Comment:
do we still need SUBMIT_PRE_PAYMENT

Reply:
yes, as it is used by migration script, will be removed in another migration PR

Reply:
@dharmendrak will fix migration.